tests/Security/Http/Authenticator/Passport/TwoFactorPassportTest.php

```diff
@@ -148,4 +150,39 @@ public function checkIfCompletelyResolved_allBadgesResolved_doNothing(): void
 
         $this->passport->checkIfCompletelyResolved();
     }
+
+    /**
+     * @test
+     */
+    public function testGetUserInstanceOf(): void
```

Comment:
Please change it to the naming pattern `[method]_[scenario]_[expectedResult]` for all test cases.